packages/mds-audit/api.ts

```diff
@@ -188,12 +204,18 @@ function api(app: express.Express): express.Express {
             })
 
             // Create the audit start event
+            const provider_event_id = res.locals.provider_event && res.locals.provider_event.id
```

Comment:
I am so looking forward to the `?.` operator in TS 3.7!!

Reply:
My thoughts exactly...